scripts/release/publish.mjs

```diff
@@ -18,6 +19,8 @@ export async function publish_handler(mode, options) {
 		options.tag
 	} --no-git-checks`;
 	const version = await getLastVersion(root);
+	core.setOutput('version', version);
```

Comment:
it's not run in action can it call @action/core?

Reply:
Do we have scenarios that not call it from actions? I can use `CI=true` to make sure it only runs in CI env, but what it dose is only write to stdout by `process.stdout.write`, do you think it's necessary to use `CI=true`?